src/components/contact/index.css

```diff
@@ -2,6 +2,14 @@
   width: 100%;
 }
 
+h1 {
```

Comment:
do you want this to affect all p and h1 on every page?

Reply:
yes, we wanted to do that